scripts/build.sh

```diff
@@ -436,7 +436,7 @@ function publish_package()
         done
         #*************************************************************************************************************#
 
-        newtonsoft=$TP_PACKAGES_DIR/newtonsoft.json/9.0.1/lib/netstandard1.0/Newtonsoft.Json.dll
+        newtonsoft=$TP_PACKAGES_DIR/newtonsoft.json/13.0.1/lib/netstandard1.0/Newtonsoft.Json.dll
```

Comment:
Since we're reading the version from Versions.props in build.ps1, can we do it here as well? This file has samples.

Reply:
done.

Reply:
You said done but I still see the version used here.